helm/robusta/values.yaml

```diff
@@ -188,3 +188,12 @@ kube-prometheus-stack:
             resources:
               requests:
                 storage: 100Gi
+  prometheus-node-exporter:
```

Comment:
One last comment I forgot to add in the review: can you check all the other Kubernetes resources in the `kube-prometheus-stack` Helm chart and make sure we didn't forget to set/unset limits/requests for any of those?